versionedRouter.js

```diff
@@ -425,6 +425,10 @@ async function routerHandleDest(ctx) {
   try {
     const { input } = ctx.request.body;
     destType = ctx.request.body.destType;
+    const routerDestHandler = getDestHandler("v0", destType);
+    input[0].metadata = routerDestHandler.processMetadataForRouter
+      ? routerDestHandler.processMetadataForRouter(input[0])
+      : input[0].metadata;
```

Comment:
This should be called for all the processed events. Once the router transformation is complete for all the transformed event present in `listOutput` each metadata should be updated.

Reply:
Updated it.